contributing-tutorials/chaos-mesh/module-introduce.md

```diff
@@ -0,0 +1,76 @@
+## 模块介绍
+
+整个 chaos-mesh 醒目，可以拆分成以下几个大块。这个章节主要介绍 chaos-mesh 的各个大块，以及各个模块的主要负责人。
+
+
+### Controllers   
+
+Controllers 模块为 Chaos Mesh 与 Kubernetes 交互模块，主要包括两部分，一部分为 CRD 结构定义，另一部分主要为接收 Kubernetes 事件以及对各类 Chaos 对象的调度与管理。
+
+#### 代码目录：
+
+- controllers 
+- api/v1alpha1
+
+负责人：@cwen0 @Yangkeao 
```

Comment:
It would be great that you can add some description about how to contact these authors on slack  channel.